docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -1,139 +1,147 @@
-- Depthwise: ScaleShift, PReLU
-- FakeQuantize
-
-> **NOTE**: You can have any number and order of simple layers.
-
-A combination of a convolution layer and simple layers results in a single fused layer called 
-*Convolution*:
-
-![conv_simple_01]
-
-
-### Fusing Pooling and FakeQuantize Layers
-
-A combination of Pooling and FakeQuantize layers results in a single fused layer called *Pooling*:  
-
-![pooling_fakequant_01]
+[Hello Query Device C++ Sample](../../../samples/cpp/hello_query_device/README.md) can be used to print out supported data types for all detected devices.
+  
+## Supported features
 
-### Fusing FullyConnected and Activation Layers
+### Multi-device execution
+If a machine has OpenVINO supported devices other than CPU (for example integrated GPU), then any supported model can be executed on CPU and all the other devices simultaneously.
+This can be achieved by specifying `"MULTI:CPU,GPU.0"` as a target device in case of simultaneous usage of CPU and GPU.
 
-A combination of FullyConnected and Activation layers results in a single fused layer called 
-*FullyConnected*:
+@snippet snippets/cpu/compile_model.cpp compile_model_multi
 
-![fullyconnected_activation_01]
+See [Multi-device execution page](../multi_device.md) for more details.
 
+### Multi-stream execution
+If either `ov::num_streams(n_streams)` with `n_streams > 1` or `ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)` property is set for CPU plugin,
+then multiple streams are created for the model. In case of CPU plugin each stream has its own host thread which means that incoming infer requests can be processed simultaneously.
+Each stream is pinned to its own group of physical cores with respect to NUMA nodes physical memory usage to minimize overhead on data transfer between NUMA nodes.
 
-### Fusing Convolution and Depthwise Convolution Layers Grouped with Simple Layers
+See [optimization guide](@ref openvino_docs_deployment_optimization_guide_dldt_optimization_guide) for more details.
 
-> **NOTE**: This pattern is possible only on CPUs with support of Streaming SIMD Extensions 4.2 
-> (SSE 4.2) and Intel AVX2 Instruction Set Architecture (ISA).
+> **NOTE**: When it comes to latency, one needs to keep in mind that running only one stream on multi-socket platform may introduce additional overheads on data transfer between NUMA nodes.
+> In that case it is better to run inference on one socket (please see [deployment optimization guide (additional configurations)](@ref openvino_docs_deployment_optimization_guide_dldt_optimization_guide_additional) for details).
 
-A combination of a group of a Convolution (or Binary Convolution) layer and simple layers and a group of a Depthwise Convolution
-layer and simple layers results in a single layer called *Convolution* (or *Binary Convolution*):
-> **NOTE**: Depthwise convolution layers should have the same values for the `group`, input channels, and output channels parameters.
+### Dynamic shapes
```

Comment:
I would propose to restructure a little bit:

- Paragraph on functional coverage: we should say that CPU plugin provides full functional support in terms of opset coverage and which exact dimension could be dynamic. Then we should add a note that dynamic rank is not supported.
- Paragraph on performance constraints: we should say that dynamic shapes support introduce some overheads and may limit internal runtime optimizations, so if input data shape is not changed between inference calls it is recommended to use model with static shape or reshape existing model to static shape to get best performance [optional: + code snitppet].
Otherwise we should recommend to provide shape with upper bounds, because [motivation] + [code snippet]